autoextract_poet/items.py

```diff
@@ -2,17 +2,28 @@
 
 import attr
 
+from autoextract_poet.util import attr_prepare
+
 
 @attr.s(auto_attribs=True, slots=True)
 class Item:
 
     @classmethod
     def from_dict(cls, item: Optional[Dict]):
+        """
+        Read an item from a dictionary, ignoring unknown attributes for
+        backwards compatibility
```

Comment:
I think we can go this way, but maybe it'd be nice to save unknown attributes to a special attribute on an item. That's possible to do it later, the current approach is fine. With such featurre you can implement page objects which do "fix this field, return all other fields as-is" even if there are new fields in the schema.

Reply:
I was thinking something like that. Let me see if I have time to implement it today.

Reply:
I'm sort of time, so better let's address this in the future as you suggest.